parm/soca/obs/obs_list.yaml

```diff
@@ -15,6 +15,8 @@ observers:
 # Ice concentration
 - !INC ${MARINE_OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${MARINE_OBS_YAML_DIR}/icec_amsr2_south.yaml
+#- !INC ${MARINE_OBS_YAML_DIR}/icec_viirs_n20_l2_north.yaml
```

Comment:
Did you mean to comment this out? Same question for "south"

Reply:
Not part of `CI` yet, we will need to comment out when we have to use them-